classes/Divisions.php

```diff
@@ -161,9 +161,9 @@ public function getRecentDebatesWithDivisions($number = 20, $majors = null) {
             $debate_gid = fix_gid_from_db($debate['debate_gid']);
             $anchor = '';
             if ($debate['c'] == 1) {
-                $anchor = '#g' . gid_to_anchor(fix_gid_from_db($debate['gid']));
+                $anchor = '#g' . gid_to_anchor(fix_gid_from_db($debate['debate_gid']));
```

Comment:
I think this needs to remain gid and the "min(h.gid)" needs an "as gid" adding, rather than switch to a different gid.